test/base-01-errors-startup.js

```diff
@@ -0,0 +1,28 @@
+var tape = require('tape');
+var mjAPI = require('../lib/main.js');
+
+// NOTE this test must first (in tape) to test the startup phase (hence the filename contains "01")
```

Comment:
I think if you use `mjAPI.start()` after the config in your callback, you should be able to put this test anywhere, as the should restart MathJax (by obtaining a whole new jsdom window).

Reply:
copy that.

Reply:
Would it be best to restart after resetting the config at the end?